docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -239,13 +241,6 @@
   See [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-windows)
   to learn more.
 
-  The following example executes the contents of the file specified by `user_data_file`:
```

Comment:
This got removed cause it was put it in a generated doc file rather than the source document, I will fix this before releasing most likely in a seperate PR